core/domain/exp_services.py

```diff
@@ -292,8 +372,15 @@ def get_exploration_titles_and_categories(exp_ids):
 
 
 def _get_exploration_summaries_from_models(exp_summary_models):
-    """Given an iterable of ExpSummaryModel instances, create a dict containing
-    corresponding exploration summary domain objects, keyed by id.
+    """Given an iterable of ExpSummaryModel instances, create a dict
+    containing corresponding exploration summary domain objects, 
+    keyed by id.
+
+    Args:
+        exp_summary_models: instances of ExplorationSummaryModel instances. 
```

Comment:
say the type:

exp_summary_models: list. A list of ExplorationSummaryModel instances.